### PR TITLE
chore: better formatting on issue template 

### PR DESCRIPTION
**Pull request by efstathiosntonas**
## Summary

use `bug-report.yml` instead of `bug-report.md` for better formatting and flow, the .md one was hard to follow when creating a new issue 😓 

[this is how the template will look like from now on](https://github.com/jpudysz/react-native-unistyles/blob/170a6085c5b3b85c1c668224f0732cba36e25c37/.github/ISSUE_TEMPLATE/bug-report.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        1.
+        2.
+        3.
+    validations:
+      required: true
+
+  - type: input
+    id: repro
+    attributes:
+      label: Snack or a link to a repository
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:43]: I would add (optional): `Snack or a link to a repository (optional)`

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Additional information
+
+        Providing as much information as possible greatly helps us with reproducting the issues.
+
+  - type: dropdown
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:88]: It can be removed

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    id: runtime
+    attributes:
+      label: JavaScript runtime
+      description: What runtime is your application using?
+      options:
+        - JSC
+        - Hermes
+        - V8
+
+  - type: dropdown
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:98]: This one too, it doesn't matter at this point

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    id: workflow
+    attributes:
+      label: Workflow
+      description: How is your application managed?
+      options:
+        - React Native
+        - Expo Go
+        - Expo Dev Client
+
+  - type: dropdown
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:108]: Same here

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    id: architecture
+    attributes:
+      label: Architecture
+      description: What React Native architecture your application is running on? Currently, the default architecture on React Native is Paper so if you haven't changed it in your application select this option.
+      options:
+        - Paper (Old Architecture)
+        - Fabric (New Architecture)
+
+  - type: dropdown
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:117]: Same here, maybe in the future when I add compiler

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+    id: build-type
+    attributes:
+      label: Build type
+      description: What mode your application is running?
+      multiple: true
+      options:
+        - Debug mode
+        - Release mode
+
+  - type: dropdown
```
Comment (jpudysz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:127]: same here, works the same everywhere

**Review by jpudysz — CHANGES_REQUESTED** [review]
Thank you for this improvement, I would remove some fields, as they don't matter with the current implementation

**Comment by efstathiosntonas** [comment]
@jpudysz will apply the changes tomorrow!

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 7]
```diff
@@ -0,0 +1,146 @@
+name: Bug report
+description: Report an issue with react-native-unistyles here.
+labels: 'Needs review'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Before you proceed:
```
[the remark this answers is not in the document]
Comment (efstathiosntonas) [reply .github/ISSUE_TEMPLATE/bug-report.yml:10]: @jpudysz which line you mean? L8 or L10?

[the remark this answers is not in the document]
Comment (jpudysz) [reply .github/ISSUE_TEMPLATE/bug-report.yml:10]: entire Line 10

**Comment by efstathiosntonas** [comment]
@jpudysz I think we're fine now: https://github.com/jpudysz/react-native-unistyles/blob/90868101a9d13f91869d146b049a8c4b65653811/.github/ISSUE_TEMPLATE/bug-report.yml

**Comment by jpudysz** [comment]
I will update it, no worries. Thanks for spotting it
